cmd/kubelet/app/server.go

```diff
@@ -96,9 +96,12 @@ const (
 
 // NewKubeletCommand creates a *cobra.Command object with default parameters
 func NewKubeletCommand() *cobra.Command {
-	// ignore the error, as this is just for generating docs and the like
-	s, _ := options.NewKubeletServer()
-	s.AddFlags(pflag.CommandLine)
+	kubeletServer, err := options.NewKubeletServer()
```

Comment:
Why can't we construct this inside Run, just like it's constructed in main() today?

Reply:
You need to use it to later add flags.